src/utils/generateTerms.ts

```diff
@@ -0,0 +1,12 @@
+// executed by node.js
+
+import strings from '../locales/strings'
+import { writeFileSync } from 'fs'
+const en = {}
+
+Object.keys(strings)
+  .sort()
+  .map(key => (en[strings[key]] = strings[key]))
+
+const path = __dirname + '/../locales/en.json'
+writeFileSync(path, JSON.stringify(en), { flag: 'w' })
```

Comment:
`JSON.stringify(en, null, 2)` would pretty print the JSON with 2 spaces per indent. It would be nicer for git, so when strings are changed we can see what actually changed.